pymc3/backends/base.py

```diff
@@ -40,6 +44,9 @@ def __init__(self, name, model=None, vars=None):
                            for var, value in var_values}
         self.chain = None
 
+    def values(self, point):
+        return (point[v] for v in self.varnames)
+
     ## Sampling methods
```

Comment:
good point. I wasn't quite sure what to call it. I'll think about it a bit more. It should probably have a leading underscore too